app/uk/gov/hmrc/servicedependencies/service/SlugDependenciesService.scala

```diff
@@ -44,12 +47,22 @@ class SlugDependenciesService @Inject()(
   def curatedLibrariesOfSlug(name: String, flag: SlugInfoFlag): Future[Option[List[Dependency]]] =
     slugInfoService.getSlugInfo(name, flag).flatMap {
       case None           => Future.successful(None)
-      case Some(slugInfo) => curatedLibrariesOfSlugInfo(slugInfo).map(Some.apply)
+      case Some(slugInfo) => latestVersionRepository
+        .getAllEntries
+        .flatMap(latestVersions => curatedLibrariesOfSlugInfo(slugInfo, latestVersions).map(Some.apply))
+    }
+
+  /*
+ * We may want to evolve the model - but for this initial version we reuse the existing Dependency definition.
+ */
+  def curatedLibrariesOfSlug(name: String, flag: SlugInfoFlag, latestVersions: Seq[MongoLatestVersion]): Future[Option[List[Dependency]]] =
+    slugInfoService.getSlugInfo(name, flag).flatMap {
+      case None           => Future.successful(None)
+      case Some(slugInfo) => curatedLibrariesOfSlugInfo(slugInfo, latestVersions).map(Some.apply)
     }
 
-  private def curatedLibrariesOfSlugInfo(slugInfo: SlugInfo): Future[List[Dependency]] =
+  private def curatedLibrariesOfSlugInfo(slugInfo: SlugInfo, latestVersions: Seq[MongoLatestVersion]): Future[List[Dependency]] =
```

Comment:
are we using the `latestVersions` param? Should we? (it defining latestVersions below L70)